src/test/kotlin/uk/gov/justice/digital/hmpps/sendlegalmailtoprisonsapi/barcode/BarcodeEventServiceTest.kt

```diff
@@ -8,34 +8,36 @@ import com.nhaarman.mockitokotlin2.verify
 import com.nhaarman.mockitokotlin2.whenever
 import org.assertj.core.api.Assertions.assertThat
 import org.assertj.core.api.Assertions.assertThatThrownBy
+import org.junit.jupiter.api.BeforeEach
 import org.junit.jupiter.api.Nested
 import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertDoesNotThrow
+import uk.gov.justice.digital.hmpps.sendlegalmailtoprisonsapi.barcode.BarcodeStatus.CHECKED
+import uk.gov.justice.digital.hmpps.sendlegalmailtoprisonsapi.barcode.BarcodeStatus.CREATED
 import uk.gov.justice.digital.hmpps.sendlegalmailtoprisonsapi.config.Duplicate
+import uk.gov.justice.digital.hmpps.sendlegalmailtoprisonsapi.config.Expired
 import uk.gov.justice.digital.hmpps.sendlegalmailtoprisonsapi.config.ValidationException
+import java.time.Duration
 import java.time.Instant
 import java.time.temporal.ChronoUnit
 import javax.persistence.EntityNotFoundException
 
 class BarcodeEventServiceTest {
 
   private val barcodeEventRepository = mock<BarcodeEventRepository>()
-  private val barcodeEventService = BarcodeEventService(barcodeEventRepository)
+  private val barcodeConfig = mock<BarcodeConfig>()
+  private val barcodeEventService = BarcodeEventService(barcodeEventRepository, barcodeConfig)
```

Comment:
Not that I have any strong feelings about this (other than how I've always done things in the past), but do we not use the mockito annotations? Like this:
```
@ExtendWith(MockitoExtension::class)
class BarcodeEventServiceTest {

  @Mock
  private lateinit var barcodeEventRepository: BarcodeEventRepository

  @Mock
  private lateinit var barcodeConfig: BarcodeConfig

  @InjectMocks
  private lateinit var barcodeEventService: BarcodeEventService
```

Reply:
It's become a convention in HMPPS not to use InjectMocks... I think because there are some edge cases where it doesn't work very well (maybe something to do with constructor defaults?) but tbh I can't remember the detail.